db/seeds.rb

```diff
@@ -175,6 +175,13 @@ def setup_application_instances(application, application_instances)
   end
 end
 
+BundleInstance.find_each do |bundle_instance|
+  bundle_instance.applications.each do |app|
```

Comment:
Even though this only looks through all of the applications associated with bundle_instance? If we have more than 1000 applications associated with a bundle_instance we might have bigger problems.